src/test/java/nl/tudelft/view/StartScreenStateTest.java

```diff
@@ -46,17 +49,33 @@ public void setUp() {
         Mockito.when(mockedResources.getTitleScreenBackground()).thenReturn(mockedImage);
 
         state = new StartScreenState(mockedResources);
+
         mockedController = Mockito.mock(StartScreenStateController.class);
         state.setController(mockedController);
+
+        mockedMouseOverArea = Mockito.mock(MouseOverArea.class);
+        Mockito.when(mockedController.createPlayer1Button(Mockito.any())).thenReturn(
+                mockedMouseOverArea);
+        Mockito.when(mockedController.createPlayer2Button(Mockito.any())).thenReturn(
+                mockedMouseOverArea);
+        Mockito.when(mockedController.createOptionsButton(Mockito.any())).thenReturn(
+                mockedMouseOverArea);
+        Mockito.when(mockedController.createHighscoresButton(Mockito.any())).thenReturn(
+                mockedMouseOverArea);
+        Mockito.when(mockedController.createQuitButton(Mockito.any())).thenReturn(
+                mockedMouseOverArea);
+
         mockedFont = Mockito.mock(TrueTypeFont.class);
         Mockito.when(mockedResources.createFont(new Font("Verdana", Font.BOLD, 36), true))
```

Comment:
new Font("Verdana", Font.BOLD, 36), true)
can be replaced by "any()" to improve readability, as you need to return a mocked font for every font which will/can be created. If you want to be certain you only return the mocked font when this particular font is created you should keep it as is.